index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+      }
+
+      body {
+        display: grid;
+        place-items: center;
+
+        background: #191210;
+      }
+
+      .one,
+      .two {
+        border-radius: 50%;
+
+        display: grid;
+        place-items: center;
+      }
+
+      .one {
+        width: 140px;
+        height: 140px;
+        background: #eca03d;
+
+        position: relative;
+      }
+
+      .two {
+        width: 100px;
+        height: 100px;
+        background: #191210;
+      }
+
+      .three {
+        width: 50px;
+        height: 50px;
+        background: #84271c;
+        border-radius: 50%;
+      }
+
+      .mini,
+      .mini-one {
+        width: 100px;
+        height: 100px;
+
+        border-radius: 50%;
+
+        display: grid;
+        place-items: center;
+
+        position: absolute;
+
+        z-index: -1;
+
+        background: #191210;
+      }
+
+      .mini {
+        border-right: #191210 20px solid;
+        border-bottom: #191210 20px solid;
+        border-left: #eca03d 20px solid;
+        border-top: #eca03d 20px solid;
+
+        right: 0;
+
+        transform: translate(80%) rotate(45deg);
+      }
+
+      .mini-one {
+        left: 0;
+
+        transform: translateX(-80%) rotate(45deg);
+
+        border-right: #eca03d 20px solid;
+        border-bottom: #eca03d 20px solid;
+        border-left: #191210 20px solid;
+        border-top: #191210 20px solid;
```

Comment:
it's better to write shorthand properly for better understanding.

Reply:
Shorthand Applied